docs/source/python/filesystems.rst

```diff
@@ -69,3 +135,66 @@ here is how you can read contents from a S3 bucket::
    >>> f = s3.open_input_stream('my-test-bucket/Dir1/File2')
    >>> f.readall()
    b'some data'
+
+
+Hadoop File System (HDFS)
+-------------------------
+
+PyArrow comes with bindings to the Hadoop File System (based on C++ bindings
+using ``libhdfs``, a JNI-based interface to the Java Hadoop client). You connect
+using the `class`:HadoopFileSystem: constructor::
```

Comment:
Looks like a markup typo in `class`.

Reply:
Ah, yes, I switched the `` ` `` and `` : `` order .. :)